packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -329,7 +329,13 @@ async function generateBase64({ file, args = {}, reporter }) {
 
   // duotone
   if (options.duotone) {
-    pipeline = await duotone(options.duotone, options.toFormat, pipeline)
+    if (options.duotone.highlight && options.duotone.shadow) {
+      pipeline = await duotone(options.duotone, options.toFormat, pipeline)
+    } else {
+      reporter.warn(
+        `To use duotone for ${file.absolutePath}, please specify duotone.highlight and duotone.shadow as the two hex colors you want to use.`
```

Comment:
Should we be more specific since the format is unique? Say something like "Please pass and object to duotone with the keys highlight and shadow with the corresponding hex values you want to use."

Reply:
Great suggestion, done!